{
  "name": "test_two",
  "status": "failed",
  "statusDetails": {
    "message": "AssertionError: assert False\n +  where False = hasattr('hello', 'check')",
    "trace": "self = <ck159890793.study_0411.test_add.TestClass object at 0x0000000004339438>\n\n    def test_two(self):\n        x = 'hello'\n>       assert hasattr(x, 'check')\nE       AssertionError: assert False\nE        +  where False = hasattr('hello', 'check')\n\ntest_add.py:19: AssertionError"
  },
  "start": 1618731491376,
  "stop": 1618731491377,
  "uuid": "5bea1da7-c921-47f4-ae5a-0430f88add1d",
  "historyId": "ca01a7838a82d639601af60696d66f7d",
  "testCaseId": "897d4338299eac7c307a12a211e1d25a",
  "fullName": "test_add.TestClass#test_two",
  "labels": [
    {
      "name": "suite",
      "value": "test_add"
    },
    {
      "name": "subSuite",
      "value": "TestClass"
    },
    {
      "name": "host",
      "value": "USER-20181110YA"
    },
    {
      "name": "thread",
      "value": "18012-MainThread"
    },
    {
      "name": "framework",
      "value": "pytest"
    },
    {
      "name": "language",
      "value": "cpython3"
    },
    {
      "name": "package",
      "value": "test_add"
    }
  ]
}